terragraph_planner/common/rf/link_budget_calculator.py

```diff
@@ -186,21 +186,20 @@ def extract_gain_from_radio_pattern(
     )
 
     # Convert deviation from (-180, 180] to [0, 360)
+    # Round value first so small negative number goes to 0 not 360
+    el_dev_360 = round(el_deviation)
     el_dev_360 = (
-        el_deviation
-        if el_deviation >= 0
-        else el_deviation + FULL_ROTATION_ANGLE
+        el_dev_360 if el_dev_360 >= 0 else el_dev_360 + int(FULL_ROTATION_ANGLE)
```

Comment:
`FULL_ROTATION_ANGLE` is converted to `int` to fix the precision issue about small negative numbers, right? Just wondering if we can make it `int` in the constant file.

Reply:
It's converted to int because the deviation is being used as an index, so el_dev_360 and az_dev_360 must be ints.

Might be possible to convert to int in constants file but it's probably used as a float everywhere else.